modules/Neo4jDataAccess.py

```diff
@@ -206,17 +206,20 @@ def __get_neo4j_graph(self, role_type):
             self.graph = None
         return self.graph
 
-    def get_from_neo(self, cypher, limit=1000):
+    def get_from_neo(self, cypher: str, limit=1000, unlimited=False):
         graph = self.__get_neo4j_graph('reader')
         # If the limit isn't set in the traversal then add it
         if not re.search('LIMIT', cypher, re.IGNORECASE):
```

Comment:
We need an opt-out for not forcing adding LIMIT, like if someone calls APOC stuff

Reply:
ex:

```
if not (limit is None) and not re.search('LIMIT', cypher, re.IGNORECASE):
    cypher = cypher + " LIMIT " + str(limit)
```

Reply:
My bad.  now I see what you mean.  Let me fix that before we merge.